S/squashfs_tools/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "squashfs_tools"
+version = v"4.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/plougher/squashfs-tools.git", "c570c6188811088b12ffdd9665487a2960c997a0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd squashfs-tools/squashfs-tools/
+sed -i 's|#XZ_SUPPORT = 1|XZ_SUPPORT = 1|' Makefile
+sed -i 's|#LZO_SUPPORT = 1|LZO_SUPPORT = 1|' Makefile
+sed -i 's|#LZ4_SUPPORT = 1|LZ4_SUPPORT = 1|' Makefile
+sed -i 's|#ZSTD_SUPPORT = 1|ZSTD_SUPPORT = 1|' Makefile
+make
+cp mksquashfs unsquashfs ${bindir}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(p -> Sys.islinux(p) && p.libc == :glibc, supported_platforms())
```

Comment:
```suggestion
platforms = filter(p -> Sys.islinux(p) && libc(p) == "glibc", supported_platforms())
```

Reply:
Oh, that's how it should work :)

Reply:
Still nothing gets built... And locally the new line gives empty `platforms` as well.

Reply:
It was built here: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=6986&view=results.  No sure why it wasn't rebuilt the second time though.  Try pushing a new change

Reply:
Oh, because you did a force-push without any change: https://github.com/JuliaPackaging/Yggdrasil/compare/36a74b492cfc31060af3d1ab74c9fe0930d8b620..9b24f7df157dea0e796a5e3a0281553841720f2a

Reply:
So everything is actually built?

Reply:
To make sure

> Try pushing a new change